OutOfSchool/OutOfSchool.AuthCommon/Validators/CustomPasswordRules.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace OutOfSchool.AuthCommon.Validators;
+public static class CustomPasswordRules
+{
+    public static bool IsValidPassword(string? password)
```

Comment:
(Code design) Could you make this class and method non-static and implementing some interface? This makes related Attribute code (uncovered now) testing easier.

Reply:
@VadymLevkovskyi I've made the requested changes. Thank you!